object.go

```diff
@@ -630,6 +634,30 @@ func (obs *obs) Delete(name string) error {
 	return obs.js.purgeStream(obs.stream, &StreamPurgeRequest{Subject: chunkSubj})
 }
 
+func publishMeta(info *ObjectInfo, js JetStreamContext) error {
+	// Prepare the meta message
+	metaSubj := fmt.Sprintf(objMetaPreTmpl, info.Bucket, encodeName(info.ObjectMeta.Name))
+	mm := NewMsg(metaSubj)
+	mm.Header.Set(MsgRollup, MsgRollupSubject)
+
+	info.ModTime = time.Time{} // We don't store this, just do it after publish
```

Comment:
This is unnecessary - `time.Time` is a struct, and a zero value for a struct is an empty struct (in this case `time.Time{}`. 

BTW, I wonder whether that's valid - empty time value will get marshaled to `0001-01-01T00:00:00Z` - is `ModTime` required in the meta message or could we simply add `omitempty` on the struct tag in `ObjectInfo{}`? (this will make `json.Marshal()` not add this field in output JSON if it's empty)